juju/application.py

```diff
@@ -83,7 +84,7 @@ def status_message(self):
 
         result = await app_facade.AddUnits(
             application=self.name,
-            placement=to,
+            placement=[parse_placement(to)],
```

Comment:
@tvansteenburgh If you take a look at the docstring for AddUnits (line 7575 in _client.py -- it's too big to link), it wants a Sequence for placement. Apparently, like AddMachines, it expects that we might bundle up multiple placement commands in one call.